pl_examples/models/lightning_template.py

```diff
@@ -46,7 +46,7 @@ def __init__(self,
                  out_features: int = 10,
                  hidden_dim: int = 1000,
                  **kwargs
-                 ) -> 'LightningTemplateModel':
+                 ):
```

Comment:
it returns an object of this type, so why to remove?

Reply:
no, we never have any return statements in `__init__`, self is the object.

Reply:
the return type of init is always `None`
https://docs.python.org/3/reference/datamodel.html#object.__init__

Reply:
would you prefer nothing or `-> None`?

Reply:
rather blank then `None`, using `None` for me means getting really `None` :)